samples/tab-sso/nodejs/README.md

```diff
@@ -148,7 +148,7 @@ Compared to the Hello World sample, this app has four additional routes:
 1. `/ssoDemo` renders the tab UI. 
     * This is the tab called `Auth Tab` in personal app inside Teams. The purpose of this page is primarily to execute the `auth.js` file that handles initiates the authentication flow.
     * This tab can also be added to Teams channels
-2. `/auth/token` does not render anything but instead is the server-side route for initiating the [on-behalf-of flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-oauth2-on-behalf-of-flow). 
+2. `/getProfile` does not render anything but instead is the server-side route for initiating the [on-behalf-of flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-oauth2-on-behalf-of-flow). 
```

Comment:
The README references this documentation (1. Create an [AAD application](https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/authentication/auth-aad-sso#1-create-your-aad-application-in-azure) in Azure.) but that documentation says "Implicit grant is not required for tab SSO." which is not correct. Without that this sample fails to work. So either remove the references to this documentation from the sample or update the documentation to clarify that for OBO flow this is required.

Reply:
Tried without implicit grant and the OBO flow worked fine.